simple_learn/classifiers/simple_classifier.py

```diff
@@ -127,41 +128,50 @@ def fit(self, train_x, train_y, folds=3):
         folds : int, optional
             The number of folds for cross validation
         """
-
-        estimators = all_estimators(type_filter="classifier")
-        for name, ClassifierClass in estimators:
-            if name in model_param_map:
-                param_grid = model_param_map[name]
-                grid_clf = GridSearchCV(
-                    ClassifierClass(),
-                    param_grid,
-                    cv=folds,
-                    scoring="accuracy",
-                    verbose=0,
-                    n_jobs=-1,
-                )
-                start = time.time()
-                try:
-                    grid_clf.fit(train_x, train_y)
-                except BaseException as error:
-                    self.failed_models.append(name)
-                    self.logger.warning(f"{name} failed due to, Error : {error}.")
-                    continue
-                end = time.time()
-                if grid_clf.best_score_ > self.metrics.get("Training Accuracy", 0.0):
-                    self.metrics["Training Accuracy"] = grid_clf.best_score_
-                    pred_y = grid_clf.predict(train_x)
-                    self.metrics["Jaccard Score"] = jaccard_score(
-                        train_y, pred_y, average="macro"
-                    )
-                    self.metrics["F1 Score"] = f1_score(
-                        train_y, pred_y, average="macro"
+        with tqdm(
+            total=(len(model_param_map)),
+            desc="Fitting Models",
+            unit=" Cycle",
```

Comment:
Can we change this to "Algorithms" to be more accurate

Reply:
Oh yeah sure, sorry I updated the requirements so I'll push again.